ftw/theming/icons.py

```diff
@@ -15,7 +16,9 @@ def html_tag(self):
             return WRAPPER_TEMPLATE.format(
                 cssclass=utils.get_mimetype_css_class_from_icon_path(self.url),
                 content=image_tag)
-        else:
+        elif hasattr(aq_base(self.brain), 'mime_type'):
```

Comment:
I didn't know that acquisition worked on brains?

Reply:
- brains are acquisition wrapped into catalog
- `aq_base` "safe" and also works when the object is not acquisition wrapped